src/pages/todos/components/TodoList/index.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { Dispatch, SetStateAction, useCallback } from 'react';
+
+import { TodoType } from '@pages/todos';
+import {
+  StyledTodoItem,
+  StyledTodoListItems,
+} from '@pages/todos/components/TodoList/TodoList.styles';
+import CheckBoxComponent from '@components/CheckBox';
+import ButtonComponent from '@components/Button';
+
+type TodoListProps = {
+  todos: TodoType[];
+  setTodos: Dispatch<SetStateAction<TodoType[]>>;
+};
+
+function TodoListComponent({ todos, setTodos }: TodoListProps): React.ReactElement {
+  const handleChangeStatus = useCallback(
+    (id: string) => {
+      setTodos((previousTodos) =>
+        previousTodos.map((todo_) =>
```

Comment:
```suggestion
        previousTodos.map((todo) =>
```
it's left over I guess